apps/accounts/views.py

```diff
@@ -315,3 +326,40 @@ def get_context_data(self, form, **kwargs):
             # inject data from all previous steps for rendering
             context["preview_forms"] = self._get_data_for_preview()
         return context
+
+    def _send_notification_email(self, user: User, provider_request: ProviderRequest):
+        """
+        Send notification to support staff, and the user to acknowledge their submission
+        """
+
+        current_site = get_current_site(self.request)
+
+        request_path = reverse("provider_request_detail", args=[provider_request.id])
+
+        link_to_verification_request = f"{self.request.scheme}://{current_site.domain}/{request_path}"
```

Comment:
`reverse` will return an absolute path, so `request_path` already starts with a `/`. Then I think that the URL needs to look like this: `f"{self.request.scheme}://{current_site.domain}{request_path}"`. Could you add an assertion for the URL included in the email in the test that you wrote to verify this please?